### DISCUSSION TITLE
Conversion of MONAI label app for inference to a MONAI bundle

### DISCUSSION THREAD

**Discussion by deepakri201** [Project-MONAI/MONAI#5894] [Q&A]
Hi, 

We(@ccosmin97 and I) are interested in running inference using AI segmentation models from MONAI Label App. However, running basic inference using MONAI Label seems to be a complicated process to scale it up over a large number of patients. We are planning to convert the MONAI Label vertebrae segmentation models to MONAI bundles. We're wondering if anyone has tried similar use cases of scaling up inference using a MONAI Label App model, either converting to MONAI bundles or through perhaps the MONAI Deploy app SDK. 

Our plans and current progress are captured in [this page](https://projectweek.na-mic.org/PW38_2023_GranCanaria/Projects/MONAILabel2bundle/), and we will post updates in this discussion as well as we make progress and have anything we learned to share.

Thank you!

**Comment by fedorov** [comment]
To add to the above, one of the reasons this is important is because we want to be able to streamline the process of batch analyzing data in [NCI Imaging Data Commons](https://github.com/ImagingDataCommons/IDC-Examples/tree/master/notebooks/getting_started) using the models trained with MONAI-Label.

**Comment by Nic-Ma** [comment]
Add @SachidanandAlle , @tangy5 , @wyli for more discussion.

Thanks.

**Comment by tangy5** [comment]
Hi @deepakri201 @fedorov , thanks for creating this issue. Let us know if there are other questions, we can discuss here. Thanks so much.

**Comment by deepakri201** [comment]
Linking this discussion: https://github.com/Project-MONAI/MONAILabel/issues/1269
